src/client/containers/authentification/authentification.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import FormComponent from '../formComponent/formComponent';
+import { formFieldsInfo } from '../../helpers/constants';
+import './authentification.css';
+
+class Authentification extends Component {
+    state = {
+        isMember: true
+    }
+    
+    onLinkClick = () => {
+        this.setState(()=>{
```

Comment:
Because new value of "isMember" prop depends on previous value, and I want to be sure that state, that l am trying to change, already is up to date in this moment.